packages/components/src/LightBox/LightBox.test.tsx

```diff
@@ -1,135 +1,192 @@
 import React from "react";
-import { fireEvent, render } from "@testing-library/react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import {
+  BREAKPOINT_SIZES,
+  mockViewportWidth,
+} from "@jobber/hooks/useBreakpoints";
 import { LightBox } from ".";
 
-test("Lightbox opens and shows the image", () => {
-  const title = "Dis be a title";
-  const caption = "Dis be a caption 🎉";
-  const handleClose = jest.fn();
-
-  const { queryByText } = render(
-    <LightBox
-      open={true}
-      images={[
-        {
-          title: title,
-          caption: caption,
-          url: "https://i.imgur.com/6Jcfgnp.jpg",
-        },
-      ]}
-      onRequestClose={handleClose}
-    />,
-  );
-
-  expect(queryByText(title)).toBeInstanceOf(HTMLElement);
-  expect(queryByText(caption)).toBeInstanceOf(HTMLElement);
-});
-
-test("lightbox doesn't show up", () => {
-  const title = "Dis be a title";
-  const caption = "Dis be a caption 🎉";
-  const handleClose = jest.fn();
-
-  const { queryByText } = render(
-    <LightBox
-      open={false}
-      images={[
-        {
-          title: title,
-          caption: caption,
-          url: "https://i.imgur.com/6Jcfgnp.jpg",
-        },
-      ]}
-      onRequestClose={handleClose}
-    />,
-  );
-
-  expect(queryByText(title)).toBeNull();
-  expect(queryByText(caption)).toBeNull();
-});
+const { setViewportWidth } = mockViewportWidth();
 
-test("lightbox closes when user clicks close", () => {
-  const title = "Dis be a title";
-  const caption = "Dis be a caption 🎉";
+describe("LightBox", () => {
+  beforeEach(() => {
+    setViewportWidth(BREAKPOINT_SIZES.lg);
+  });
+  test("Lightbox opens and shows the image", () => {
```

Comment:
nitpick: since we're nested in a `Lightbox` `describe` all these `test`s don't really need to be prefixed with `Lightbox` anymore since when they run, it'll read as: "Lightbox Lightbox opens and shows the image"

personally I appreciate it when tests read as a somewhat-normal sentence that can more or less be understood from reading the message